src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -300,40 +296,39 @@ public void apply(final VariantContext variant,
                       final ReadsContext readsContext,
                       final ReferenceContext referenceContext,
                       final FeatureContext featureContext) {
-        final Collection<VariantContext> vcs = featureContext.getValues(getDrivingVariantsFeatureInput());
 
         final Collection<VariantContext> otherVCs = featureContext.getValues(supportVariants);
 
         final int missing = supportVariants.size() - otherVCs.size();
 
-        for ( final VariantContext vc : vcs ) {
-            final VariantContext vc_familyPriors;
-            final VariantContext vc_bothPriors;
+        final VariantContext vc_familyPriors;
+        final VariantContext vc_bothPriors;
 
-            //do family priors first (if applicable)
-            final VariantContextBuilder builder = new VariantContextBuilder(vc);
-            //only compute family priors for biallelelic sites
-            if (!skipFamilyPriors && vc.isBiallelic()){
-                final GenotypesContext gc = famUtils.calculatePosteriorGLs(vc);
-                builder.genotypes(gc);
-            }
+        //do family priors first (if applicable)
+        final VariantContextBuilder builder = new VariantContextBuilder(variant);
+        //only compute family priors for biallelelic sites
+        if (!skipFamilyPriors && variant.isBiallelic()){
+            final GenotypesContext gc = famUtils.calculatePosteriorGLs(variant);
+            builder.genotypes(gc);
+        }
+        VariantContextUtils.calculateChromosomeCounts(builder, false);
+        vc_familyPriors = builder.make();
+
+        if (!skipPopulationPriors) {
+            vc_bothPriors = PosteriorProbabilitiesUtils.calculatePosteriorProbs(vc_familyPriors, otherVCs, missing * numRefIfMissing, globalPrior, !ignoreInputSamples, defaultToAC, useACoff);
+        } else {
+            final VariantContextBuilder builder2 = new VariantContextBuilder(vc_familyPriors);
             VariantContextUtils.calculateChromosomeCounts(builder, false);
```

Comment:
Oops, this should probably be builder2, right?  All this time and no one ever said anything.

Reply:
It definitely looks like it should be.  It's effectively the same thing, but it's very confusing and if anyone ever make changes it could lead to problems... I might start a separate PR to address that.

Reply:
wait, what am I talking about... the thing that gets output should be the the result of modifying builder2 and we modify builder.  That seems super wrong.

Reply:
no, i'm back to "it's dumb but fine"

Reply:
new pr to fix this https://github.com/broadinstitute/gatk/pull/4431